automap/src/comm_layer/pcp_pmp_common/linux_specific.rs

```diff
@@ -153,17 +197,35 @@ mod tests {
         168.63.129.16 via 10.1.0.1 dev eth0 proto dhcp src 10.1.0.84 metric 100\n\
         169.254.169.254 via 10.1.0.1 dev eth0 proto dhcp src 10.1.0.84 metric 100\n\
         172.17.0.0/16 dev docker0 proto kernel scope link src 172.17.0.1 linkdown";
-        let reg_for_any_other_line = ip_route_regex();
+        let regex = ip_route_regex();
```

Comment:
At least one of the lines above should probably begin with `default via` to fully exercise the regular expression.